app/repositories/package.scala

```diff
@@ -119,7 +121,9 @@ package object repositories {
 
     applicationAssessmentScoresRepository.collection.indexesManager.create(Index(Seq(("applicationId", Ascending)), unique = true)),
 
-    assessorAvailabilityRepository.collection.indexesManager.create(Index(Seq(("userId", Ascending)), unique = true))
+    assessorAvailabilityRepository.collection.indexesManager.create(Index(Seq(("userId", Ascending)), unique = true)),
+
+    assessmentEventsRepository.collection.indexesManager.create(Index(Seq(("eventType", Ascending), ("date", Ascending)), unique = false))
```

Comment:
We will want to do queries on location and venue in the future, might be a good idea to add these indices too